lib/clean-evm-node.js

```diff
@@ -16,6 +16,21 @@ const buildDisplayInfo = k => {
     stack_kast
   } = getInfo(k);
 
+  let kcell = kast.get(k, 'k');
+
+  const reduceKSequence = term => {
+    if(term.node == "KApply" && term.label == "#KSequence") {
+      return [term.args[0]].concat(reduceKSequence(term.args[1]))
```

Comment:
I think it could be more readable if the elements of `term.args` were saved in variables with descriptive names.